test/e2e/upgrades/apps/deployments.go

```diff
@@ -140,47 +134,9 @@ func (t *DeploymentUpgradeTest) Test(f *framework.Framework, done <-chan struct{
 
 	By(fmt.Sprintf("Waiting for deployment %q to complete adoption", deployment.Name))
 	framework.ExpectNoError(framework.WaitForDeploymentStatus(c, deployment))
-
```

Comment:
You're no longer checking replicasets. Therefore the log in L126 needs to be updated.